docs/source/style_guide.md

```diff
@@ -6,9 +6,9 @@ While this style guide will also refer to best practices using ReStructured Text
 
 **When in doubt, use the docs themselves for guidance on how to format things.**
 
-* [For RST formatting](http://hyperledger-fabric.readthedocs.io/en/release-1.4/channel_update_tutorial.html).
+* [For RST formatting](http://hyperledger-fabric.readthedocs.io/en/latest/channel_update_tutorial.html).
```

Comment:
Instead of latest here, I would recommend using {BRANCH} here since this content exists in all the branches.